src/main/java/de/tum/in/www1/artemis/domain/exam/monitoring/ExamAction.java

```diff
@@ -13,6 +16,13 @@
 /**
  * Defines an action performed by a student during an exam.
  */
+@Entity
+@Table(name = "exam_action")
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@DiscriminatorColumn(name = "type", discriminatorType = DiscriminatorType.STRING)
+@DiscriminatorOptions(force = true)
+@org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
```

Comment:
You should add an import for the annotation

Reply:
Oh good catch :D